### DISCUSSION TITLE
ModifySerialization method from VerifySettings class not found on the newer versions

### DISCUSSION THREAD

**Discussion by jonathanperis** [VerifyTests/Verify#956] [General]
Hello!

I'm working on a migration project that uses the Verify **v14.8.1** and we've tried to upgrade to the evergreen version (at the moment of this writing, **v20.6.0**) and the project uses the ModifySerialization method from the VerifySettings class to do some things, but in this new version it looks like we don't have that method anymore. 

![image](https://github.com/VerifyTests/Verify/assets/13967942/30d75a0a-9dc0-4517-86f7-bea6d338db0e)

We have some type of replacement of that method, or some way that we can do the same goal on another way?

**Comment by SimonCropp** [comment]
https://github.com/VerifyTests/Verify/blob/main/docs/serializer-settings.md
